### PR TITLE
Add configuration argument, improve log output

### PR DESCRIPTION
**Pull request by djMax**
#111 allow building configurations other than release

### CODE REVIEW

**Comment by neonichu** [comment]
LGTM, but could you please also update the relevant tests (see <https://travis-ci.org/CocoaPods/cocoapods-packager/builds/111611113#L408> ff).

**Comment by djMax** [comment]
Updated tests

File: lib/pod/command/package.rb [hunk 1]
```diff
@@ -34,6 +35,8 @@ def initialize(argv)
         subspecs = argv.option('subspecs')
         @subspecs = subspecs.split(',') unless subspecs.nil?
 
+        @config = argv.option('configuration') || 'Release'
```
Comment (segiddins) [inline lib/pod/command/package.rb]: `arvg.options('configuration', 'Release')`

**Comment by djMax** [comment]
Failure seems to just be timeout?

**Comment by neonichu** [comment]
Yep, triggered a rebuild.

**Comment by neonichu** [comment]
Thanks for the contribution :tada: (and sorry for taking so long to merge it).

**Pull request merged**